src/transport/GroupSession.h

```diff
@@ -66,6 +66,8 @@ class IncomingGroupSession : public Session
     const char * GetSessionTypeString() const override { return "incoming group"; };
 #endif
 
+    ScopedNodeId GetPeer() const override { return ScopedNodeId(mSourceNodeId, GetFabricIndex()); }
```

Comment:
`GetPeerScopedNodeId` ? Thinking that `Peer` by itself is reasonably vague as we have `PeerId` defined.